template-helpers/hbs-helpers.js

```diff
@@ -54,6 +54,22 @@ function recruitmentBanner(args) {
   return `<div class="recruitment-banner"><a id="recruitment-banner" href="${AppConstants.RECRUITMENT_BANNER_LINK}" hidden target="_blank" rel="noopener noreferrer" data-ga-link="" data-event-category="Recruitment" data-event-label="${escapeHtmlAttributeChars(AppConstants.RECRUITMENT_BANNER_TEXT)}">${AppConstants.RECRUITMENT_BANNER_TEXT}</a></div>`;
 }
 
+function showCsatBanner(args) {
+  const signupDate = args.data.root.req.session.user?.created_at;
+
+  if (!signupDate) return; // don't show if user is not logged in or not signed up
+
+  if (args.data.root.req.cookies.csatHidden) return; // don't show if user closed banner
+
+  if (Date.now() - Date.parse(signupDate) < 604800000) return; // don't show if sign-up is less than 7 days old
```

Comment:
nit: I find `7 * 24 * 60 * 60 * 1000` a bit more obvious for timestamp delta values.

Reply:
True.  I threw it in the comment, to be even _more_ obvious/annoying 😅 Don't have much of a preference either way